ONSdigital/rm-case-service
# Remove bi/new b case migration script

# Motivation and Context
Because of difficulties communicating with multiple databases in the prod/preprod environments we will now be creating a temporary table in the case service database and executing sql directly against the databases rather than leveraging a script for this.

# What has changed
Two scripts are included:
1. `SELECT` script to create temporary table required for B case creation
2. `INSERT` script to create new B case rows where appropriate

A `README.md` file is also included to explain the procedure for using these scripts

# How to test?
Set up some test data. You'll have to create some initial cases using either the acceptance tests or by manually executing collection exercises and enrolling some respondents against these. 
Since we now do create B cases you'll also have to manually delete some of the existing ones to test the scripts.

Follow the instructions in `README.md` to create the missing B cases and check that this has been successful in the database

# Links
[Trello](https://trello.com/c/qxS2f9RO/281-update-b-case-creation-script-data-migration)
